.github/workflows/validate.yml

```diff
@@ -0,0 +1,27 @@
+name: Validate workflows
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-22.04
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.9'
+
+    - name: Setup engines
+      run: | 
+        curl -o requirements.txt "https://raw.githubusercontent.com/dockstore/dockstore/1.13.0-beta.3/dockstore-webservice/src/main/resources/requirements/1.13.0/requirements3.txt"
+        pip3 install --user -r requirements.txt
+
+    - run: sudo apt-get install -y moreutils
+
+    - name: Validate all the things 
```

Comment:
This only validates the CWL. Consider adding the WDL and NFL equivalents too. Not sure about nextflow but for WDL the command would be `java -jar womtool.jar validate [whatever.wdl]` which will return 0 plus the text "Success!" if it passes.

Reply:
Strictly speaking validating the WDL isn't fully necessary -- when `dockstore --script  workflow launch --local-entry checker.wdl  --json md5sum.wdl.json` runs, validation will occur first. But if we go to the trouble of validating the CWL, why not go to that trouble for everything?

Reply:
Adding wdl validation